### ISSUE TITLE
Is Reef-Pi Dead??? I see tons of pull requests and none being committed

### ISSUE THREAD

**Issue by piwi3910** [reef-pi/reef-pi#2234]
Like the title said?
Is Reef-Pi Dead??? I see tons of pull requests and none being committed
Do you need maintainers to help?

**Comment by danmrossi** [comment]
Think it’s easier said than done. I keep getting folks myself telling me its not dead and I’d help too if I could understand the code lol but yeah I think its dead, I haven’t seen any info from the main guy that writes this app saying he’s getting back into it anytime soon.

**Comment by hectorespert** [comment]
Hi @ranjib

Seeing that the PRs and Issues are accumulating, I wanted to propose if I could help with that in some way.

**Comment by danmrossi** [comment]
Hi Hector, just thinking out loud here and I don’t fully understand all the inner GitHub workings but if you don’t get a response from Ranjib then if you still want to help maybe have to fork it and advise us here, do all the changes /release etc and then when Ranjib is back can then merge all the changes into the main repository or something?

Sent from my iPhone

On 22 Dec 2024, at 2:49 am, Hector Espert ***@***.***> wrote:

﻿

Hi @ranjib<https://github.com/ranjib>

Seeing that the PRs and Issues are accumulating, I wanted to propose if I could help with that in some way.

**Comment by ranjib** [comment]
@hectorespert yes, need some help with triaging. If you have some bandwidth, I can add you as maintainer to triage/merge PRs/  release

**Comment by ranjib** [comment]
@danmrossi <3 , any specific feature/bug you want to help with or in general? 
I'll get some time now I think. But please continue the effort with fork if there's any substantial changes and this one is not moving. Meanwhile, I'll see if I can pull in @hectorespert  :-) , to assist with the triage.

**Comment by danmrossi** [comment]
Hi @ranjib just mainly looking for current raspberry pi os support with the new kernel mainly at this point, I’ve worked around a lot of the quirks by mixture of the reef-pi home assistant addon and mqtt :)
Sent from my iPhone

On 23 Dec 2024, at 11:56 am, Ranjib Dey ***@***.***> wrote:

﻿

@hectorespert<https://github.com/hectorespert> yes, need some help with triaging. If you have some bandwidth, I can add you as maintainer to triage/merge PRs/ release

**Comment by danmrossi** [comment]
> @danmrossi <3 , any specific feature/bug you want to help with or in general? I'll get some time now I think. But please continue the effort with fork if there's any substantial changes and this one is not moving. Meanwhile, I'll see if I can pull in @hectorespert :-) , to assist with the triage.

Thinking about it i think it would be nice if you could duplicate the lights section, call it pumps as i am using the lights section to control lights and pumps via 0-10v control

**Comment by ranjib** [comment]
yes, actually duplication will clutter the backend and frontend code equally and becomes maintenance burden. I was thinking of making a more generic variable voltage outlet and  let user to declare any pwm profile , timer etc.. that way user can use the same module for controlling light as well as pump. I want to do the same for analog input as well , I,e ph sensor, temp sensor all to be combined in a single module. they have similar issues.

btw, did you find a externally controllable pump (0-10v) ? over the time I became less interested in controlling lights and pumps and most of them have their own controller and the basic ones (like kessil and LED bricks) can already be controlled with existing reef-pi capabilities

**Comment by danmrossi** [comment]
Yeah or just call the lights section instead maybe just PWM? So it can be for anything

Yes I’m controlling 0-10v kessils and for pumps some jebao gyres with a coralbox Aqlink a1 0-10v converter…

<https://m.fish-street.com/aqlink-a1-wifi-jebao-maxspect-controller>
AQLink A1 for Apex Controller for Crossflow Gyre DCS DCT Pump<https://m.fish-street.com/aqlink-a1-wifi-jebao-maxspect-controller>
fish-street.com<https://m.fish-street.com/aqlink-a1-wifi-jebao-maxspect-controller>
[X]<https://m.fish-street.com/aqlink-a1-wifi-jebao-maxspect-controller>

Lists the known models that can connect but a lot more can be with plug changes etc: https://www.fish-street.com/video/a1manual.pdf

Which I then ultimately control from passing thru the controls from reefpi to home assistant and I can create my own power profiles, wave profiles etc

Sent from my iPhone

On 24 Dec 2024, at 2:54 am, Ranjib Dey ***@***.***> wrote:

﻿

yes, actually duplication will clutter the backend and frontend code equally and becomes maintenance burden. I was thinking of making a more generic variable voltage outlet and let user to declare any pwm profile , timer etc.. that way user can use the same module for controlling light as well as pump. I want to do the same for analog input as well , I,e ph sensor, temp sensor all to be combined in a single module. they have similar issues.

btw, did you find a externally controllable pump (0-10v) ? over the time I became less interested in controlling lights and pumps and most of them have their own controller and the basic ones (like kessil and LED bricks) can already be controlled with existing reef-pi capabilities

**Comment by hectorespert** [comment]
> @hectorespert yes, need some help with triaging. If you have some bandwidth, I can add you as maintainer to triage/merge PRs/ release

I will have some spare time to help with that during the next 6 months.
